source/beamline/BioXAS/BioXASBeamline.cpp

```diff
@@ -338,6 +338,19 @@ CLSValves* BioXASBeamline::valves() const
 	return result;
 }
 
+AMXRFDetector *BioXASBeamline::xrfDetector() const
```

Comment:
This implementation works for now, as there is only one Ge detector on Side and Main. But that won't always be the case. I'm not sure this implementation makes sense for this future case.